index.html

```diff
@@ -153,6 +153,10 @@ <h2>The Club</h2>
           <td><a href="https://codeberg.org/kiesel-js/kiesel">Kiesel</a></td>
           <td>a JavaScript engine in Zig, for fun</td>
         </tr>
+        <tr>
+          <td><a href="https://adrianmato.com/pewpew">github-pewpew</a></td>
+          <td>a CLI tool to pew pew your unused repos from GitHub, for fun</td>
```

Comment:
We try to keep it to 3-5 words, so how about:

```suggestion
          <td>a CLI tool to delete repos, for fun</td>
```

...or similar?